KlipTok.de-DE.json

```diff
@@ -2,43 +2,115 @@
-	"NoProfilePicAvailable": "Kein Profilbild verfügbar"
+	"LastClipAdded": "Neuster Clip:",
+	"NoProfilePicAvailable": "Kein Profilbild verfügbar",
+
+
+	// Layout
+	"PublicBeta": "PUBLIC BETA",
+	"LoggedOutFromKlipTokAndTwitch": "Von KlipTok und Twitch ausgeloggt",
+	"LoginToKlipTokWithTwitch": "Mit Twitch bei KlipTon anmelden",
+	"AuthenticationIsNoLongerValid": "Deine Authentifizierung mit Twitch und KlipTok ist nicht mehr gültig. Bitte melde dich mit Twitch an, um fortzufahren.",
+	"Loading": "Laden...",
+
+	// Errors
+	"ErrorWhileLoading": "Fehler beim Laden.",
+	"RetryButton": "Erneut versuchen",
+
+	// Follows page
+	"TwitchChannelsIFollow": "Kanäle, denen ich folge",
+	"NotLoggedIn": "Bitte melde dich an, um Kanäle anzuzeigen, denen du folgst.",
+
+	// Home Page
+	"DiscoverPopularClips": "Beliebte Clips entdecken",
+	"DiscoverClips": "Clips entdecken",
+	"InsertCoin": "Insert Coin",
+	"MyFavoriteChannels": "Meine Favoriten",
+	"Favorites": "Favoriten",
+	"ClipsILike": "Clips, die ich mag",
+	"LikedClips": "Clips, die ich mag",
+	"UserProfile": "Profil",
+
+	// Liked Clips
+	"NoLikedClips": "Du hast noch bei keinem Clip 'Gefällt mir' geklickt!",
+	"TryLikeButtonTemplate": "Klicke unter einem Clip auf das Herz {0} and we'll report it here for you so you can easily find it again in the future",  // {0} ist das Herz Icon
+
+	// User Profile
+	"DoesNotKeepSeparateUserInformation": "KlipTok speichert keine Nutzerinformation abseits von Twitch, and wurde als Service entwickelt, um Twitch Clips zu erweitern.",
+	"AutoPlayVideosOnScroll": "Videos beim Scrollen automatisch abspielen",
+	"IndexClipsOfMyChannel": "Meine Clips indizieren",
+	"OptedOutDateTemplate": "Automatische Indizierung abgewählt am: {0}", // {0} ist das Datum des Opt-outs
+	"SaveMyChanges": "Änderungen speichern",
+	"SyncFollowersWithKlipTok":"Kanäle, denen ich folge mit, KlipTop synchronisieren",
+	"MutedChannels": "Stummgeschaltete Streamer",
+	"Channel": "Kanal",
+	"MutedUntil": "Stummgeschaltet bis",
+	"RightToForget": "Recht auf Vergessenwerden",
+	"DangerZone": "[ Gefahrenzone ]", // This is a label indicating the items beneath it make destructive changes to your record on the application 
+	"RightToForgetDescription": "Du hast das Recht, dass KlipTok alle Informationen, die Du geteilt hast, löscht. Mit einem Klick auf diesen Link bestätigst Du, dass KlipTok alle deine Kommentare, Likes und Clips entfernt.",
+	"RemoveMyContentFromKlipTok": "Meine Informationen und Inhalte von KlipTok löschen",
+	"ForgottenAsOfTemplate": "Daten gelöscht am: {0}", // {0} ist das Datum, an dem die Datenlöschung veranlasst wurde
+
+	// Single Clip Page
+	"MissingClipError": "KlipTok kennt diesen Clip nicht. Entweder existiert er nicht, oder der Twitch Kanal hat die Indizirung auf KlipTok abgewählt.",
+	"LikedBy": "Gefällt:",
+	"UserLikedTimeTemplate": "{0} gefällt dieser Clip", // {0} ist der Displayname des Nutzers, dem der Clip gefällt, {1} ist der relative Zeitstempel
```

Comment:
This is a bit of an ugly construct in german. For now I have opted to omit the timestampt and actually translate `like`, as I have done with the rest of the translations.
I could use an anglicism, which would make this sound a lot more natural with the timestamp, but I'm honestly not sure which would be preferred by users.